pkg/kubelet/cm/dra/types.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dra
+
+import (
+	v1 "k8s.io/api/core/v1"
+	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
+)
+
+// Manager manages all the DRA resource plugins running on a node.
+type Manager interface {
+	// PrepareResources prepares resources for a container in a pod.
+	// It communicates with the DRA resource plugin to prepare resources and
+	// returns resource info to trigger CDI injection on the runtime side.
```

Comment:
> s/returns/return

PrepareResources **prepares** ... it **communicates** ... and **returns**
I thought all of the verbs should be in the same tense, no?

Reply:
I guess it depends on how you parse the sentence., but as its constructed it reads more naturally as
> it "communicates" "to prepare and return"
rather than 
> it "communicates" "to prepare", it "returns"

Reply:
Would it look more understandable if we split it to two sentences, e.g.
```
// PrepareResources prepares resources for a container in a pod.
// It communicates with the DRA resource plugin to prepare resources.
// It returns resource info to trigger CDI injection on the runtime side.
```

Reply:
It's fine as is, not worth nitpicking over

Reply:
Resolved